dts/arm/nordic/nrf51822.dtsi

```diff
@@ -133,6 +134,18 @@
 			label = "SPI_1";
 		};
 
+		/* nRF TEMP Peripheral:
+		 * .. Default enabled,
+		 * .. Interrupt priority:
+		 * .. .. Range: 0-1, Default: 1
+		 */
+		temp: temp@4000C000 {
+			compatible = "nordic,nrf-temp";
+			reg = <0x4000C000 0x1000>;
```

Comment:
The addresses should be in lowercase. See https://github.com/zephyrproject-rtos/zephyr/commit/523932ccb0a6dadbef0b273150ee3e80b906e915.

Reply:
(Y) Just so used to writing hex symbols as uppercase. Will change it.